lib/views/pr-comments-view.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Point, Range} from 'atom';
+
+import Marker from '../atom/marker';
+import MarkerLayer from '../atom/marker-layer';
+import Decoration from '../atom/decoration';
+
+import Timeago from './timeago';
+
+export default class PullRequestCommentsView extends React.Component {
+
+  render() {
+    if (this.props.reviews) {
+      return this.props.reviews.nodes.map(review => {
+        return review.comments.nodes.map(comment => {
+          if (comment.position !== null) {
+            const filePatch = this.props.multiFilePatch.getFilePatchByPath(comment.path);
+            const commentStartPoint = filePatch.getStartRange().start.translate(new Point(comment.position, 0));
+            const range = new Range(commentStartPoint, commentStartPoint);
+            const author = comment.author;
+            return (
+              <Marker key={`pr-comment-${comment.id}`} bufferRange={range} invalidate="never">
+                <Decoration type="block" className="github-PrComment">
```

Comment:
FYI: I changed the class name from `github-FilePatchView-prComments-controlBlock` to `github-PrComment`. To reflect [make `PullRequestCommentsView` into its own component](https://github.com/atom/github/pull/1856/commits/b341c5dd9c1cca132a260a28fd5bce55abc541ac). Styles are in `/styles/pr-comment.less`.